packages/@react-stately/dialog/src/useDialogTriggerState.ts

```diff
@@ -14,13 +14,22 @@ import {DialogTriggerProps} from '@react-types/dialog';
 import {useControlledState} from '@react-stately/utils';
 
 export interface DialogTriggerState {
+  /** Whether the dialog is currently open. */
```

Comment:
In the DialogTrigger docs pull all of the code file descriptions have Dialog with a capital letter as it is referencing a component. Should be consistent, either changed here or there.

Reply:
Actually if I remember right from past pulls it's always with a capital D

Reply:
So, I agree in the React Spectrum docs because it's referring to the Dialog component. But inside stately, it's more referring to the concept of a dialog/dialog trigger rather than a specific implementation.